furniture_tick.cpp

```diff
@@ -49,7 +50,7 @@ static void handleBoulder(Position pos, WFurniture furniture) {
             pos.globalMessage(PlayerMessage("The boulder starts rolling.", MessagePriority::CRITICAL));
             pos.unseenMessage(PlayerMessage("You hear a heavy boulder rolling.", MessagePriority::CRITICAL));
             CHECK(!pos.getCreature());
-            pos.addCreature(CreatureFactory::getRollingBoulder(furniture->getTribe(), direction), 0_visible);
+            pos.addCreature(CreatureFactory::getRollingBoulder(TribeId::getMonster(), direction), 0_visible);
```

Comment:
Please change it to TribeId::getHostile(). With getMonster() there may be some attackers that won't fear it.